toc/rfc/rfc-draft-use-github-packages-container-registry-instead-of-dockerhub.md

```diff
@@ -0,0 +1,37 @@
+# Meta
+[meta]: #meta
+- Name: Distribute Public Community Images using Github Packages Container Registry instead of DockerHub
+- Start Date: 2024-04-18
+- Author(s): @tcdowney
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (fill in with PR link after you submit it)
+
+
+## Summary
+
+Move away from DockerHub and distribute official Cloud Foundry container images using the [Github Packages](https://github.com/features/packages) [Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry). This RFC is primarily concerned with public container images that are published for the Cloud Foundry community.
+
+## Terminology
+
+* **Community Image -** A public container image that is intended to be consumed by members of the Cloud Foundry community.  Some examples are the `cloudfoundry/cli` and `cloudfoundry/cflinuxfs4` images that are published by the CLI and Buildpacks teams.
```

Comment:
Do we have a list of community images that require a more formal migration process? E.g. in the CF docs?